opensfm/src/bundle.h

```diff
@@ -51,8 +56,33 @@ struct BAPerspectiveCamera : public BACamera{
   void SetK2(double v) { parameters[BA_CAMERA_K2] = v; }
 };
 
+struct BABrownPerspectiveCamera : public BACamera{
+  double parameters[BA_CAMERA_BROWN_DISTORTION_NUM_PARAMS];
+  double focal_prior;
+  double k1_prior;
+  double k2_prior;
+  double p1_prior;
+  double p2_prior;
+  double k3_prior;
+
+  BACameraType type() { return BA_PERSPECTIVE_CAMERA; }
```

Comment:
That should be BA_BROWN_PERSPECTIVE_CAMERA. I think that must be the reason why bundle was failing and you were not getting good results @bdholt1